lib/ansible/module_utils/aws/urls.py

```diff
@@ -0,0 +1,166 @@
+# (C) Unknown?
```

Comment:
https://docs.ansible.com/ansible/latest/dev_guide/developing_modules_documenting.html#copyright

Reply:
This has been corrected now